fetch/api/basic/request-forbidden-headers.js

```diff
@@ -0,0 +1,49 @@
+if (this.document === undefined) {
+  importScripts("/resources/testharness.js");
+  importScripts("../resources/utils.js");
+}
+
+function requestForbiddenHeaders(desc, forbiddenHeaders) {
+  var url = RESOURCES_DIR + "inspect-headers.py";
+  var requestInit = {"headers": forbiddenHeaders}
+  var urlParameters = "?headers=" + Object.keys(forbiddenHeaders).join("|");
+
+  promise_test(function(test){
+    return fetch(url + urlParameters, requestInit).then(function(resp) {
+      assert_equals(resp.status, 200, "HTTP status is 200");
+      assert_equals(resp.type , "basic", "Response's type is basic");
+      for (var header in forbiddenHeaders)
+        assert_not_equals(resp.headers.get("x-request-" + header), forbiddenHeaders[header], header + " has not the value we defined");
+    });
+  }, desc);
+}
+
+//origin header policy still under discussion for no-cors fetches
```

Comment:
I am not sure what this comment was pointing.
Let's remove it.

As of no-cors header tests, coverage might be missing and could be added to mode-no-cors.html or a separated test file.